src/js/hooks/usePrevious.js

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from 'react';
+
+export default function usePrevious(value) {
```

Comment:
What is it for? Is is used anywhere?

Reply:
I also forgot to delete it after refactoring `if`. Should we consider keeping it anyway, as this hook might be useful in the future or should I delete it?